e2e-tests/cypress/support/appeal-submission-tasklist/confirmUserPresentedWithProvidePlanningApplicationNumber.js

```diff
@@ -1,4 +1,4 @@
 module.exports = () => {
-  cy.url().should('contain','/appellant-submission/application-number');
+  cy.url().should('contain','/appeal-householder-decision/application-number');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks